GiniVision/Classes/Core/Screens/Camera/Camera.swift

```diff
@@ -44,7 +44,7 @@ final class Camera: NSObject, CameraProtocol {
         return videoDeviceInput?.device.hasFlash ?? false
         #endif
     }()
-    var isFlashOn: Bool = true
+    lazy var isFlashOn: Bool = giniConfiguration.flashOnByDefault
```

Comment:
I'm not sure you really benefit from the laziness. If you just put it there to delay the setting, it's maybe not the best solution.